src/main/helpers.js

```diff
@@ -295,6 +295,25 @@ export function readableStream(data) {
   s.push(null)
   return s
 }
+
+// Process metadata to insert appropriate value to `content-type` attribute
+export function insertContentType(metaData, filePath) {
+  // first convert case sensitive `content-type` attributes to lowercase
+  var newMetadata = Object.assign({}, metaData)
+  // check if content-type attribute present in metadata
+  var found = false
+  for (var key in newMetadata) {
+    if (key.toLowerCase() === 'content-type') {
+      found = true
```

Comment:
You can break once you have set it to true.

Reply:
Thanks @kannappanr, I just applied the change.